tests/integrations/import/sst_service.rs

```diff
@@ -101,28 +101,41 @@ fn test_ingest_sst() {
     meta.set_region_id(ctx.get_region_id());
     meta.set_region_epoch(ctx.get_region_epoch().clone());
     send_upload_sst(&import, &meta, &data).unwrap();
-    // Cann't upload the same file again.
+    // Can't upload the same file again.
     assert!(send_upload_sst(&import, &meta, &data).is_err());
 
     ingest.set_sst(meta.clone());
     let resp = import.ingest(&ingest).unwrap();
     assert!(!resp.has_error());
+}
+
+#[test]
+fn test_ingest_sst_without_crc32() {
+    fail::cfg("raft_before_delete_sst", "sleep(1000)").unwrap();
```

Comment:
If you want to make a `fail_point` wait a while for some event or signal, you can set a callback in test code and make it wait for notifying, which is a  new feature of `fail_point`. See more details in https://github.com/tikv/fail-rs/pull/50/files